app/css/custom.scss

```diff
@@ -871,4 +947,4 @@ blockquote {
       }
     }
   }
-}
+}
```

Comment:
But there is a structure. Look at the directories within `app/css` to get a sense of what I meant. Looking forward to those updates :)